modules/caddyhttp/encode/encode.go

```diff
@@ -206,28 +203,33 @@ func (rw *responseWriter) Flush() {
 		// to rw.Write (see bug in #4314)
 		return
 	}
-	rw.ResponseWriterWrapper.Flush()
+	rw.HTTPInterfaces.Flush()
```

Comment:
```suggestion
	rw.Flush()
```

I think this makes more sense when it reads like this, personally; but that's just my preference :)

Reply:
It will just use all the stacks because of recursion 😄.

Reply:
Oh! Silly me, you're right.

Reply:
That's the sort of thing I would suggest deserves a code comment just in case one of us forgets the reason here and tries to simplify it

Reply:
@francislavoie My editor actually gives me a warning! I think it's a new feature of gopls or govet or something.